client/state/data-layer/wpcom/akismet-key/index.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+	AKISMET_KEY_REQUEST,
+	AKISMET_KEY_RECIEVE,
+	AKISMET_KEY_REQUEST_FAILURE,
+} from 'calypso/state/action-types';
+import { registerHandlers } from 'calypso/state/data-layer/handler-registry';
+import { http } from 'calypso/state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'calypso/state/data-layer/wpcom-http/utils';
+
+const fetchAkismetAPIKey = ( action: object ) => {
```

Comment:
I think there's a general guideline not to use the `object` type in TypeScript in calypso. If you can, we should provide the actual type, or `unknown`.